tests/test_bootstrap.py

```diff
@@ -1478,18 +1478,19 @@ def test_run(self, mock_runtime_client, mock_handle_event_request):
         "awslambdaric.bootstrap.LambdaLoggerHandler",
         Mock(side_effect=Exception("Boom!")),
     )
-    @patch("awslambdaric.bootstrap.build_fault_result", MagicMock())
+    @patch("awslambdaric.bootstrap.build_fault_result")
     @patch("awslambdaric.bootstrap.log_error", MagicMock())
     @patch("awslambdaric.bootstrap.LambdaRuntimeClient", MagicMock())
     @patch("awslambdaric.bootstrap.sys")
-    def test_run_exception(self, mock_sys):
+    def test_run_exception(self, mock_sys, mock_build_fault_result):
         class TestException(Exception):
             pass
 
         expected_app_root = "/tmp/test/app_root"
         expected_handler = "app.my_test_handler"
         expected_lambda_runtime_api_addr = "test_addr"
 
+        mock_build_fault_result.return_value = {}
```

Comment:
We need to mock `build_fault_result()` to return value because newer `simplejson` will throw an error when it tries to JSON encode a `MagicMock` type.
```
E           TypeError: _asdict() must return a dict, not MagicMock
```

https://github.com/aws/aws-lambda-python-runtime-interface-client/blob/056aa365b6b986b0f413d031f054598064df603e/awslambdaric/bootstrap.py#L485-L488